src/app/app.component.html

```diff
@@ -1,8 +1,13 @@
 <div class="root">
   <app-header></app-header>
   <div class="main">
-    <app-breadcrumbs></app-breadcrumbs>
-    <router-outlet></router-outlet>
+    <div *ngIf="authService.loginStatusObs | async; else loginPage">
```

Comment:
1. please, don't use services in view. 
You can just to refer on  this stream as follows:
```typescript
class SomeComponent {
    isLoggedIn$: Observeble<boolean> = this.authService.isLoggedIn
  constructor(private authService: AuthService) {}
}
```
2. `loginStatusObs` -> `loginStatus$` or `isLoggedIn$`;

Reply:
done